src/screens/TransactionConfirmationScreen.js

```diff
@@ -211,6 +243,13 @@ export default function TransactionConfirmationScreen() {
     return isDappAuthenticated(dappUrl);
   }, [dappUrl]);
 
+  const handleL2DisclaimerPress = useCallback(() => {
+    navigate(Routes.EXPLAIN_SHEET, {
+      network: network,
```

Comment:
hm I thought this was removed in a previous commit: `network` doesn't need to be passed along to Explain Sheet as a param

Reply:
Fixed